Configuration/Applications/python/Options.py

```diff
@@ -99,7 +99,7 @@
                   dest="runsScenarioForMC")
 
 parser.add_option("--runUnscheduled",
-                  help="Enable unscheduled mode",
+                  help="Automatically convert configuration to run unscheduled the EDProducers/EDFilters that were scheduled",
```

Comment:
Yes. Before it set the allowUnscheduled flag, did the conversion, and called the special customise functions. The difference is only that now it does not set the allowUnscheduled flag in the options ParameterSet.